src/battle_util.c

```diff
@@ -11082,56 +11079,50 @@ bool32 IsGen6ExpShareEnabled(void)
     return FlagGet(I_EXP_SHARE_FLAG);
 }
 
-bool32 MoveHasMoveEffect(u32 move, u32 moveEffect)
-{
-    u32 i;
-    for (i = 0; i < gMovesInfo[move].numAdditionalEffects; i++)
-    {
-        if (gMovesInfo[move].additionalEffects[i].moveEffect == moveEffect
-            && gMovesInfo[move].additionalEffects[i].self == FALSE)
-            return TRUE;
-    }
+#define R_CHECK_MOVE_EFFECT(...) RECURSIVELY(R_FOR_EACH(CHECK_MOVE_EFFECT, __VA_ARGS__))
+#define CHECK_MOVE_EFFECT(_condition) && effects[i]._condition
+
+/* Quick way of checking if a move has move effects with match a few
+comma-separated conditions. Each condition has to check a field of AdditionalEffect. */
+#define RETURN_HAS_MOVE_ADDITIONAL_EFFECT(condition1, ...)           \
```

Comment:
```suggestion
#define RETURN_HAS_MOVE_ADDITIONAL_EFFECT(condition1, ...)          \
```

Reply:
What am I supposed to be seeing here? 🤔 
![Uploading Screenshot_20240226-205859_Edge (1).jpg…]()

Reply:
What  I supposed to be changing here?
![Screenshot_20240226-205859_Edge (1)](https://github.com/rh-hideout/pokeemerald-expansion/assets/22523466/07fce8ba-b71b-44d9-a133-606b2ebd1f1f)

Reply:
Alignment
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/8ff29ba9-164b-4ea6-90bd-05de7611c3b4)